cogboard-webapp/src/components/Widget/helpers.js

```diff
@@ -56,3 +59,11 @@ export const renderCardContent = (
     </StyledCardContent>
   );
 };
+
+const AlwaysVisibleContent = ({ type, status, content, expanded }) => {
```

Comment:
I am not sure if ** AlwaysVisibleContent** is the most fortunate name regarding the fact it can return **null**. Maybe sth like **ExpandableContent** or **ComponentSpecificAdditionalContent**/**AdditionalContent**?

Reply:
Good point, I choose ExpandableContent

Reply:
Ok, let's catch up after conference to discuss the approach. I agree that it needs some abstraction refactor.